projects/angular-showcase/src/app/business/business/business.component.ts

```diff
@@ -51,6 +53,9 @@ export class BusinessComponent implements ExampleProvider {
     },
     usermenu: {
       'usermenu-showcase': new ComponentPortal(UsermenuShowcaseComponent)
+    },
+    table: {
+      'table-showcase': new ComponentPortal(TableShowcaseComponent)
```

Comment:
It is possible to have multiple examples entries.

Reply:
I was looking for this :) I now added a few more examples and separated them here to provide better code snippet insight.